### ISSUE TITLE
ERNIE-Gram：How to use ernie-gram to predict masked ngram fragments?

### ISSUE THREAD

**Issue by buildcharger** [PaddlePaddle/ERNIE#681]
Hello, thank you very much for your contribution: ERNIE-Gram, this is very helpful.

I would like to use ERNIE-Gram for prediction of masked ngram fragments, is it possible to provide the relevant code snippet?😃

Thanks for your help!

**Comment by PROoshio** [comment]
You can refer to the codes of the MLM implementation in ERNIE1.0~

**Comment by bigprince97** [comment]
Follow the model pre-training settings, if i use coarse-grained manner to predict masked fragments,  then ngram vocab is necessary, if I use fine-grained manner, I need to add some mask tokens and set complex attention mask, use the extra mask  token to predict individual tokens. I don't know if I understand this correctly, could you help me?
